### PR TITLE
Adds error message for when ux-backend pod is down

### PR DESCRIPTION
**Pull request by bipuladh**
[Bug 2266930](https://bugzilla.redhat.com/show_bug.cgi?id=2266930)

#### Screenshot
Screenshot from 2024-05-10 15-10-39

### CODE REVIEW

**Comment by bipuladh** [comment]
/hold
Need approval for UXD and doc team

**Comment by bipuladh** [comment]
/hold cancel 
Got text approved from doc team.

File: packages/odf/components/storage-consumers/onboarding-modal.tsx [hunk 1]
```diff
@@ -59,23 +67,40 @@ export const ClientOnBoardingModal: ClientOnBoardingModalProps = ({
...
+            {!loading && error && (
+              <Alert
+                variant="danger"
+                isInline
+                title={t('Can not generate an onboarding token at the moment')}
+              >
+                <Trans>
+                  The token generation service is currently unavailable. Contact
+                  our{' '}
+                  <ExternalLink href="https://access.redhat.com/support/">
```
Comment (SanjalKatiyar) [inline packages/odf/components/storage-consumers/onboarding-modal.tsx:83]: nits: we should keep all urls under single file/directory (good for reference purpose).

**Pull request merged**